detectors/node/opentelemetry-resource-detector-azure/src/detectors/AzureAppServiceDetector.ts

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { DetectorSync, IResource, Resource } from '@opentelemetry/resources';
+
+import {
+  CloudProviderValues,
+  CloudPlatformValues,
+  SemanticResourceAttributes,
+} from '@opentelemetry/semantic-conventions';
+
+const AZURE_APP_SERVICE_STAMP_RESOURCE_ATTRIBUTE = 'azure.app.service.stamp';
+const CLOUD_RESOURCE_ID_RESOURCE_ATTRIBUTE = 'cloud.resource_id';
```

Comment:
Looks like several of these env variable names are shared between the different detectors, it would be good to have a types file with these

Reply:
Added a types file to keep these organized and avoid redundancy.